lib/tasks/export_data.rb

```diff
@@ -0,0 +1,36 @@
+module Tasks
+  class ExportData
+    def initialize(file, stdout)
+      @file = file
+      @stdout = stdout
+    end
+
+    def export_all
+      total = Reservation.count
+
+      Reservation.all.each.with_index(1) do |reservation, index|
+        file.puts(reservation.to_json)
+
+        print_progress(index, total)
+      end
+
+      stdout.puts
+    end
+
+  private
```

Comment:
The style guide says to remove the whitespace line below the `private` keyword. It's not something I tend to agree with, or that is generally seen elsewhere but there you go.